backend/config.py

```diff
@@ -8,3 +8,4 @@
 # Access the environment variables
 SECRET_KEY = os.getenv("SECRET_KEY")
 ACCESS_TOKEN_EXPIRE_MINUTES = int(os.getenv("ACCESS_TOKEN_EXPIRE_MINUTES", 30))
+ALGORITHM = os.getenv("ALGORITHM")
```

Comment:
We don't need this environment variable. See review comment for more details.

Reply:
I removed it!